rental.rb

```diff
@@ -1,12 +1,22 @@
 class Rental
-  attr_accessor :date
-  attr_reader :book, :person
+  attr_accessor :date, :book, :person
+
+  # attr_reader
```

Comment:
**Functionality of the app is perfect. However, we have some cleanup jobs to do.** 

- [ ] Kindly remove this commented code from your project :)

Reply:
This change has been done well